source/trackers.json

```diff
@@ -735,6 +777,58 @@
         "xtracloud.net": "qualcomm",
         "qualcomm.com": "qualcomm",
         "recaptcha.net": "recaptcha",
+        "aibixby.com": "samsung",
+        "lldns.net": "samsung",
+        "samsung-omc.com": "samsung",
+        "samsungapps.com": "samsung",
+        "samsungdiroute.net": "samsung",
+        "samsungdive.com": "samsung",
+        "samsungdm.com": "samsung",
+        "samsungdmroute.com": "samsung",
+        "samsungmdec.com": "samsung",
+        "samsungvisioncloud.com": "samsung",
+        "sbixby.com": "samsung",
+        "samsungosp.com": "samsung",
+        "samsungsds.com": "samsung",
+        "internetat.tv": "samsung",
+        "samsungcloud.tv": "samsung",
+        "ospserver.net": "samsung",
+        "samsungdms.net": "samsung",
+        "samsungmax.com": "samsung",
+        "samsungmobile.com": "samsung",
+        "secmobilesvc.com": "samsung",
+        "game-mode.net": "samsung",
+        "gos-gsp.io": "samsung",
+        "pavv.co.kr": "samsung",
+        "remotesamsung.com": "samsung",
+        "samsung-gamelauncher.com": "samsung",
+        "samsung.co.kr": "samsung",
+        "samsung.com": "samsung",
+        "samsung.com.cn": "samsung",
+        "samsungcloud.com": "samsung",
+        "samsungcloudcdn.com": "samsung",
+        "samsungcloudprint.com": "samsung",
+        "samsungcloudsolution.com": "samsung",
+        "samsungcloudsolution.net": "samsung",
+        "samsungelectronics.com": "samsung",
+        "samsunghealth.com": "samsung",
+        "samsungiotcloud.com": "samsung",
+        "samsungknox.com": "samsung",
+        "samsungnyc.com": "samsung",
+        "samsungotn.net": "samsung",
+        "samsungpositioning.com": "samsung",
+        "samsungqbe.com": "samsung",
+        "samsungrm.net": "samsung",
+        "samsungrs.com": "samsung",
+        "samsungsemi.com": "samsung",
+        "samsungsetup.com": "samsung",
+        "samsungusa.com": "samsung",
+        "secb2b.com": "samsung",
+        "smartthings.com": "samsung",
+        "samsungacr.com": "samsung",
+        "samsungadhub.com": "samsung",
+        "samsungads.com": "samsung",
```

Comment:
To be correct it would be better to refer domains to the relevant trackers
`samsungads.com` : `samsungads`,
`samsungmobile.com` : `amsungmobile`,
`samsungsds.com` : `samsungsds`
etc.

Reply:
Ignore [this commit](https://github.com/AdguardTeam/companiesdb/pull/186/commits/a42ad57e7a0aa9a6aeb0637d01d410de8878733a), I misunderstood the request.

This should cover everything, a few extra domains were also added. There was a few in Netify that I wasn't so sure of such as the ones starting with "www", so please let me know if they should be there or not.